ci-operator/config/openshift/odo/openshift-odo-master__v4.5.yaml

```diff
@@ -34,7 +34,7 @@ tests:
       from: oc-bin-image
       resources:
         requests:
-          cpu: "2"
```

Comment:
Isn’t this reducing the resources? From 2 to 800m? 800m means millicpus so effectively .8. Can we not reduce 2 to 800m?

Reply:
This shows that the periodic job gets 2 cpus but the CI just gets 200m, so we can keep the 2 cpu in periodic job or increase that? But definitely increase the 200m in standard jobs

Reply:
> Isn’t this reducing the resources? From 2 to 800m? 800m means millicpus so effectively .8. Can we not reduce 2 to 800m?

@girishramnani So as per our discussion on slack `800m` is enough cpu limit for our test to run, right? Isn't it be ok if we reduce the resources according to our consumption.

Reply:
I don’t want to unnecessarily change what’s already set. So let’s increase the 200m to 800m and keep the 2 cpu as is